### PR TITLE
chore: delete OAuth docs

### PR DESCRIPTION
**Pull request by rangoo94**
## Changes

- https://github.com/kubeshop/testkube/pull/6013

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: redirects.js [hunk 1]
```diff
@@ -214,13 +214,6 @@ const redirects = [
     ],
     to: "/articles/testkube-dashboard-explore",
   },
-  {
-    from: [
-      "/guides/going-to-production/authentication/oauth-cli",
-      "/guides/authentication/oauth-cli",
-    ],
-    to: "/articles/oauth-cli",
```
Comment (olensmar) [inline redirects.js:222]: should we add a redirect to some other page so any inbound links won't get a 404?

Comment (rangoo94) [reply redirects.js:222]: Good point, would https://docs.testkube.io/articles/cli be fine?

Comment (olensmar) [reply redirects.js:222]: I think so!? should we add a note there regarding this "removal" !?

Comment (rangoo94) [reply redirects.js:222]: I'm not sure if we should:
* if yes - then maybe better to even keep that page with annotation (but someone may ignore that annotation)
* if not - if someone really needs it, will rather contact us through GitHub, Slack or other way

Comment (olensmar) [reply redirects.js:222]: hm.. perhaps we can just move this doc to the legacy-docs section and add the corresponding legacy header?

Comment (rangoo94) [reply redirects.js:222]: But isn't the Legacy section for things that are deprecated, but still there?

Comment (olensmar) [reply redirects.js:222]: true - ok - in that case just remove it and add a redirect to the article you suggested 👍

**Pull request merged**